{"name":"User repository find by id failure","fullName":"TestUserFindByIDSuite/User repository find by id/TestFindByID_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727773563267,"stop":1727773563268,"uuid":"6269f058-7fd4-11ef-a987-ca15b1947338","historyId":"1aadfd8fbe48d8c85599f851ba1f2f06","testCaseId":"3103cf8174f7a9354cd44a30d01b4991","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"h19.net58.bmstu.ru"},{"name":"thread","value":"TestUserFindByIDSuite/User repository find by id/TestFindByID_Failure"},{"name":"suite","value":"User repository find by id"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727773563268,"stop":1727773563268,"parameters":[{"name":"Error","value":"sql: no rows in result set: record does not exist"},{"name":"Target","value":"record does not exist"}]}]}